manage_breast_screening/demo/views.py

```diff
@@ -0,0 +1,51 @@
+from django.contrib.auth import get_user_model, login
+from django.contrib.auth.decorators import login_not_required
+from django.db.models import Case, Q, When
+from django.shortcuts import get_object_or_404, redirect, render
+
+from .models import PERSONAS
+
+
+def next_page(request):
+    url = request.POST.get("next", "/")
+
+    if not url.startswith("/"):
+        return request.path
+
+    return url
+
+
+def get_users(current_user):
+    users = get_user_model().objects.filter(
+        username__in=(persona.username for persona in PERSONAS)
+    )
+
+    if current_user:
+        # Stick the current user at the top of the list
+        return users.annotate(
+            ordering=Case(When(Q(username=current_user.username), then=0), default=1)
+        ).order_by("ordering", "first_name")
+
+    return users.order_by("first_name")
```

Comment:
Not sure on the conventions around this, but would it be helpful to prefix these with an underscore and place them at the bottom of the file (or something along those lines) to distinguish these helper methods from the view function?